### PR TITLE
Documentation for application ids

### PR DESCRIPTION
**Pull request by karthikbr82**

### CODE REVIEW

**Review by vr000m — APPROVED** [review]
It seems we dont provide examples here, otherwise copying the examples from the other PR would make sense

File: source/includes/_api.md [hunk 1]
```diff
@@ -51,11 +53,39 @@ It provides developers a method to enable or disable certain features or functio
...
+var applicationIDs = {
+  customerID: "customer ID",
+  pbxID: "PBX ID",
+  pbxExtensionID: "PBX Extension ID",
+  fqExtensionID: "fully qualified extension ID",
+  meetingsName: "Meeting Name",
+  tenantID: "Tenant ID",
+  productName: "Product Name",
+  serverName: "Server Name",
+};
```
Comment (vr000m) [inline source/includes/_api.md]: is this done before initialize? could you add that part to this example?

File: source/includes/_api.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ var configParams = {
   applicationVersion: "app_version", // Application version specified by the developer.
   disablePrecalltest: true // disables the pre-call test, it is enabled by default.
   siteID: "siteID", // The name/ID of the site/campus from where the call/pre-call test is made.
+  applicationIDs: applicationIDs, // applicationIDs object, contains application related IDs.
```
Comment (vr000m) [inline source/includes/_api.md]: should this be called userTags instead of applicationIDs?

File: source/includes/_api.md (hunk 1 above)
Comment (karthikbr82) [reply source/includes/_api.md]: It is shown in the intitialize section, with example, the javascript blob will be in right side.

Comment (karthikbr82) [reply source/includes/_api.md]: Screenshot 2020-11-25 at 15 02 08

**Pull request merged**